feat: hide extraneous elements from swagger ui

lutostag opened the pull request:
Hey there, this is based on the previous pull request #10. I went through and set `display: none;` for a bunch of the duplicated data on the swagger side. Not sure if **everyone** would want this, but it works well for me, and wanted to share.

That makes the ui look way less cluttered and more focused imo. Looks like the following:
<img width="1478" alt="Screen Shot 2021-04-28 at 5 29 30 PM" src="https://user-images.githubusercontent.com/962625/116480635-4d6c0000-a847-11eb-8331-a16c49389c11.png">

Happy for any feedback, or if you think this is too specific to my use-case feel free to close this without any explanation. Thanks for an awesome project! 🥂 

For those wanting to just drop this right into `head.styles` on index.html or however they are serving their pages themselves they could use the following.
```css
      .opblock-summary * {
        display: none;
      }
      .opblock-description-wrapper {
        display: none;
      }
      :not(.live-responses-table).responses-table {
        display: none;
      }
      .responses-inner>div>h4 {
        display: none;
      }
      .try-out {
        display: none;
      }
      .btn.cancel {
        display: none;
      }
```

lutostag committed:
feat: hide extraneous fields from swagger by default

wll8 commented:
I don't know if it will affect similar functions.

- link: https://wll8.github.io/redoc-try/#operation/uploadFile

![image](https://user-images.githubusercontent.com/28223260/116504106-316d6a80-a8ea-11eb-9f5c-295411bfffa3.png)

lutostag committed:
fix: do not hide opblock header with auth

lutostag commented:
Good point.

I may have gone too far. I removed hiding of the opblock header, but kept the others hidden.

Rebased on master now that #10 landed. Open to other feedback or rejection on this one as it is a bit more controversial.

<img width="1475" alt="Screen Shot 2021-04-29 at 7 28 00 AM" src="https://user-images.githubusercontent.com/962625/116550710-b2157200-a8bc-11eb-83d4-98d462b60330.png">
